providers/ydb/pyproject.toml

```diff
@@ -49,17 +49,17 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
     "Topic :: System :: Monitoring",
 ]
-requires-python = ">=3.10"
+requires-python = ">=3.10,!=3.13"
 
 # The dependencies should be modified in place in the generated file.
 # Any change in the dependencies is preserved when the file is regenerated
 # Make sure to run ``breeze static-checks --type update-providers-dependencies --all-files``
 # After you modify the dependencies, and rebuild your Breeze CI image with ``breeze ci-image build``
 dependencies = [
-    "apache-airflow>=2.10.0",
-    "apache-airflow-providers-common-sql>=1.20.0",
-    "ydb>=3.18.8",
-    "ydb-dbapi>=0.1.0",
+     "apache-airflow>=2.10.0",
+     "apache-airflow-providers-common-sql>=1.20.0; python_version < \"3.13\"",
+     "ydb>=3.18.8; python_version < \"3.13\"",
+     "ydb-dbapi>=0.1.0; python_version < \"3.13\"",
```

Comment:
nit: by accident here are 5 instead of 4 spaces

Reply:
Will fix.